test/cli/remote-create/table/in/definition.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "table1",
+  "primaryKeysNames": [
+    "id"
+  ],
+  "columns": [
+    {
+      "name": "id",
+      "definition": {
+        "type": "VARCHAR"
```

Comment:
On a call, we discussed (with Petr), that the `--columns-from definition.json` should contain only columns array `[]`.

Now it is possible to enter the name of the table and primary keys via flag or via file. And invalid combinations are not handler nor tested.

For example, I enter primary keys via flag, I also enter `columns-from`.

Reply:
```
[
    {
      "name": "id",
      "definition": {
        "type": "VARCHAR"
....
```

Reply:
```
--bucket <string>
Bucket ID
--columns <string>
Comma-separated list of column names
--name <string>
Name of the table
--primary-key <string>
Comma-separated list of columns used as a primary key
```

Reply:
`--columns-from`

Reply:
`--columns-from` NOR `--columns` <- check